### PR TITLE
Lvnod 123 create about us

### PR DESCRIPTION
**Pull request by yurii-kokhalevych**
## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/browse/LVNOD-123)

## Code reviewers

- [x] @denisbodnar
- [x] @AlbaMontis
- [x] @shyyka

### Second Level Review

- [ ] @lhalam
- [ ] @4asof4ik

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo
## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=90%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -3701,9 +3701,9 @@
       "integrity": "sha1-u8qATAQE2cZAECsrZWrkzVvqNcg="
```
Comment (AlbaMontis) [inline package-lock.json]: Put this file to the gitignore.

File: app/src/containers/About/About.js [hunk 2]
```diff
@@ -1,7 +1,45 @@
...
+            gifts. Register and get the opportunity to create an event and send an 
+            invitation to the guests. Make your list of gifts, and guests can 
+            choose any of the gifts you desire, buy it individually or join forces.</p>
+          </Col>
+        </Row>
+        <Row className='show-grid'>
+          <PageHeader className='text-center'>Our Team</PageHeader>
+          {Object.keys(Names).map(param =>
+            <Col xs={3} sm={3} md={3}>
+              <Image src={Names[param]} thumbnail/>
```
Comment (AlbaMontis) [inline app/src/containers/About/About.js]: It would be better to put images into backgrounds.

File: app/src/containers/About/About.js [hunk 3]
```diff
@@ -1,7 +1,45 @@
...
+      'Yuriy Kokhalevych': require('../../../img/Yura.svg')
+    };
+    return (
+      <Grid>
+        <Row className='show-grid'>
+          <Col xs={12} sm={12} md={12}>
+            <PageHeader className='text-center'>Da-Mi</PageHeader>
+            <p>Do you plan to celebrate your birthday, wedding or other event 
+            and do not want to get 3 identical electric kettles as a gift? 
+            Then it's up to you! Our project will help you to get only the desired
```
Comment (denisbodnar) [inline app/src/containers/About/About.js]: use ```&apos;``` instead of '

**Review by denisbodnar — DISMISSED** [review]

**Review by shyyka — DISMISSED** [review]

**Review by denisbodnar — DISMISSED** [review]

File: app/src/containers/About/About.scss [hunk 4]
```diff
@@ -0,0 +1,59 @@
+@import '../../common.scss';
+
+#gift {
+  background: url('../../../img/gift.svg');
+  height: 450px;
+  max-width: 450px;
+  width: $width-100;
+  background-repeat: $background-repeat;
```
Comment (AlbaMontis) [inline app/src/containers/About/About.scss]: You can put it after url in background.

File: package-lock.json [hunk 5]
```diff
@@ -25,8 +25,7 @@
       "dependencies": {
         "acorn": {
           "version": "4.0.13",
-          "resolved": "https://registry.npmjs.org/acorn/-/acorn-4.0.13.tgz",
-          "integrity": "sha1-EFSVrlNh1pe9GVyCUZLhrX8lN4c="
+          "resolved": "https://registry.npmjs.org/acorn/-/acorn-4.0.13.tgz"
```
Comment (AlbaMontis) [inline package-lock.json]: Please remove this file and put it into gitignore.

File: app/src/containers/About/About.scss [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+  }
+
+#Maryana {
```
Comment (AlbaMontis) [inline app/src/containers/About/About.scss]: I think you can use sass map here or something like that.

**Review by shyyka — DISMISSED** [review]

**Review by denisbodnar — DISMISSED** [review]

**Review by shyyka — DISMISSED** [review]

**Review by AlbaMontis — DISMISSED** [review]

File: app/src/containers/About/About.scss [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+    width: 100%;
+  }
+}
+
+#photo-gift {
+  height: 450px;
+  max-width: 450px;
+}
+
+#photo-Maryana, #photo-Denis, #photo-Ivan,
```
Comment (4asof4ik) [inline app/src/containers/About/About.scss]: use class team

File: app/src/containers/About/About.js [hunk 8]
```diff
@@ -1,7 +1,42 @@
...
+            <p className='textAbout'>{texts.descriptionOurProgectPart1}</p> 
+            <p className='textAbout'>{texts.descriptionOurProgectPart2}</p>
+          </Col>
+        </Row>
+        <Row className='show-grid'>
+          <PageHeader className='text-center'>Our Team</PageHeader>
+          <p className='textAbout'>{texts.descriptionOurTeam}</p>
+          {Object.keys(names).map(param =>
+            <Col xs={6} sm={4} md={3} key={param}>
+              <div id={names[param]} className="container"></div>
```
Comment (4asof4ik) [inline app/src/containers/About/About.js]: use common class for all img

**Review by 4asof4ik — CHANGES_REQUESTED** [review]

File: app/src/containers/About/About.js [hunk 9]
```diff
@@ -1,7 +1,42 @@
 import React from 'react';
+import { Grid, Row, Col, PageHeader, Image } from 'react-bootstrap';
```
Comment (AlbaMontis) [inline app/src/containers/About/About.js]: Do you use Image in this file?

**Review by shyyka — DISMISSED** [review]

**Review by shyyka — DISMISSED** [review]

File: .gitignore [hunk 10]
```diff
@@ -2,3 +2,4 @@ npm-debug.log
 build
 node_modules
 PULL_REQUEST_TEMPLATE.md
+package-lock.json
```
Comment (shyyka) [inline .gitignore:5]: add blank line

**Review by shyyka — DISMISSED** [review]

**Review by AlbaMontis — DISMISSED** [review]

**Review by denisbodnar — DISMISSED** [review]

**Review by 4asof4ik — APPROVED** [review]

**Review by denisbodnar — APPROVED** [review]

**Review by AlbaMontis — APPROVED** [review]

**Pull request merged**